### PR TITLE
initial psudo datamodel

### PR DESCRIPTION
**Pull request by kordjamshidi**

### CODE REVIEW

File: saul-examples/src/main/java/edu/illinois/cs/cogcomp/saulexamples/nlp/Xml/NlpXmlReader.java [hunk 1]
```diff
@@ -47,63 +43,35 @@ public NlpXmlReader(File file) {
...
-    public List<Token> getTokens(String parentId) {
-        return getTokens(tokenTagName, parentId);
-    }
-
     public List<Token> getTokens(String tagName, String parentId) {
         return getElementList(tagName, parentId, NlpBaseElementTypes.Token);
     }
 
-    public List<Relation> getAllRelations(String tagName, String name, String firstIdProp, String secondIdProp) {
+    public List<Relation> getRelations(String tagName, String name, String firstIdProp, String secondIdProp) {
```
Comment (kordjamshidi) [inline saul-examples/src/main/java/edu/illinois/cs/cogcomp/saulexamples/nlp/Xml/NlpXmlReader.java]: So, the relations can connect only two ids? Maybe we can have a list of string here each of which can be an id?

**Comment by kordjamshidi** [comment]
it seems this an old PR, do we need it? please close if the changes are in the master already.

**Comment by Rahgooy** [comment]
we didn't merge `TaherMaster` in `master` yet. but I think this PR is very old. If we want to merge now, we can make a new PR

**Comment by kordjamshidi** [comment]
Does it have anything useful?

**Comment by Rahgooy** [comment]
Everything is included in the `TaherMaster` branch, so we can close this without losing anything.

**Pull request closed without merging**